src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
 import IndexRoute from '@routes/index.route';
 import UsersRoute from '@routes/users.route';
 import validateEnv from '@utils/validateEnv';
 
 validateEnv();
 
-const app = new App([new IndexRoute(), new UsersRoute(), new AuthRoute()]);
+const app = new App([new IndexRoute(), new UsersRoute(), new AuthRoute(), new GroupsRoute()]);
```

Comment:
To improve modularity and maintainability, consider passing dependencies instead of instantiating them internally. Here's an updated version:

```suggestion
const groupsService = new GroupsService();
const groupsController = new GroupsController(groupsService);
const groupsRoute = new GroupsRoute(groupsController);
const app = new App([new IndexRoute(), new UsersRoute(), new AuthRoute(), groupsRoute]);
```